ci/perf-tester/src/index.js

```diff
@@ -0,0 +1,203 @@
+const { setFailed, startGroup, endGroup, debug } = require('@actions/core');
+const { GitHub, context } = require('@actions/github');
+const { exec } = require('@actions/exec');
+const { getInput, runBenchmark, averageBenchmarks, toDiff, diffTable, toBool } = require('./utils.js');
+
+async function run(octokit, context, token) {
+	const { number: pull_number } = context.issue;
+
+	const pr = context.payload.pull_request;
```

Comment:
Could we have a `.prettierrc` file added to the repository and make formatting consistent here with the rest of the `.ts` and `.js` files that use 4 spaces for indentation?

Reply:
Sure! Do you think it makes sense to add a `package.json` to the root of the project so the various JS parts (IntegrationTests, Runtime, perf-tester) each use the same version of Prettier?

Reply:
I don't have a clear preference here. @kateinoigakukun WDYT?

Reply:
I think it's reasonable to put package.json on the project root.